src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/SubscriptionController.cs

```diff
@@ -160,6 +160,13 @@ private bool ValidateSubscription(Subscription eventsSubscription, out string me
                 return false;
             }
 
+            if (!string.IsNullOrEmpty(eventsSubscription.AlternativeSubjectFilter)
+                && string.IsNullOrEmpty(eventsSubscription.SubjectFilter))
+            {
+                message = "A valid subject to the authenticated is required";
```

Comment:
I'm not sure what the message is trying to explain. Should it be the same as for the check above?

Reply:
They should be the same so we can use it to wash ssn

Reply:
Updated to make it more clear